tools/add-author.js

```diff
@@ -46,7 +46,7 @@ async function fetchUserProfile() {
 		name: profile.name,
 		title: "Guest Author",
 		website: profile.blog,
-		avatar_url: profile.avatar_url,
+		avatar_url: `${profile.avatar_url.replace("private-", "").split("?")[0]}?v=4`,
```

Comment:
```suggestion
		avatar_url: `${profile.avatar_url.replace("private-", "").split("?")[0]}?v=4`,
```
We could consider adding a fallback image in case the actual image isn't found, though I'm not sure if GitHub itself provides a fallback image.

Reply:
sounds good to me, should we use the fallback image which we are using for sponsors? this one
![Screenshot 2024-12-21 170645](https://github.com/user-attachments/assets/bc18dd91-abd9-468b-8fd1-ca3f237e01aa)

Reply:
I am 👍 let's get other's opinions too.

Reply:
This fallback also looks good to me.

Reply:
Added fallback image.